packages/eas-cli/src/project/publish.ts

```diff
@@ -208,6 +210,7 @@ export async function buildBundlesAsync({
       '--dump-sourcemap',
       '--dump-assetmap',
       `--platform=${platformFlag}`,
+      ...(dev ? ['--dev'] : []),
```

Comment:
I'm not sure if this flag exists on the legacy Expo CLI

Reply:
Confirming it does! https://github.com/expo/expo-cli/blob/main/packages/expo-cli/src/commands/export/export.ts#L26